fastify.d.ts

```diff
@@ -127,6 +127,7 @@ declare namespace fastify {
     logger?: pino.LoggerOptions | boolean,
     trustProxy?: string | number | boolean | Array<string> | TrustProxyFunction,
     maxParamLength?: number,
+    querystringParser?: (str: string) => { [key: string]: string | string[] | undefined },
```

Comment:
Which case a value could be `undefined`?

Reply:
Removed the undefined, everything gets parsed into a string.